app-localize-behavior.html

```diff
@@ -185,19 +212,24 @@
     }
 
     var request = proto.__localizationCache.requests[path];
+
+    function onRequestResponse(event) {
+      this.__onRequestResponse(event, language, merge, preprocessor);
```

Comment:
Where are these parameters coming from? why would they not always be null? do you mean `this.language` etc?

Reply:
Oh I see. I would just inline it in the 2 call sites, for clarity